tests/test_bokky_poo_bahs_date-time-library.py

```diff
@@ -31,6 +33,23 @@ def test_timestamp_to_date_automated(date_time_library, start_date, end_date):
         current_date += relativedelta(days=+1)
 
 
+@given(
+    timestamp=strategy(
+        "uint256",
+        min_value=get_timestamp_from_date(2020, 1, 1),
```

Comment:
As long as we'd like to make the coverage as dense as it's possible in reasonable test time, I think it doesn't make sense to start from 2020. Start from today or at least 2022.